source/404.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<meta charset="utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<title>404</title>
+
+<!-- Google font -->
+<link href="https://fonts.googleapis.com/css?family=Nunito:400,700" rel="stylesheet">
+
+<!-- Custom stlylesheet -->
+<style type="text/css">
+   * {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+#notfound {
+  position: relative;
+  height: 100vh;
+}
+
+#notfound .notfound {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  -webkit-transform: translate(-50%, -50%);
+      -ms-transform: translate(-50%, -50%);
+          transform: translate(-50%, -50%);
+}
+
+.notfound {
+  max-width: 560px;
+  width: 100%;
+  padding-left: 160px;
+  line-height: 1.1;
+}
+
+.notfound .notfound-404 {
+  position: absolute;
+  left: 0;
+  top: 0;
+  display: inline-block;
+  width: 140px;
+  height: 140px;
+  background-image: url('https://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Home_Assistant_Logo.svg/1200px-Home_Assistant_Logo.svg.png');
+  background-size: cover;
+}
+
+.notfound .notfound-404:before {
+  content: '';
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  -webkit-transform: scale(2.4);
+      -ms-transform: scale(2.4);
+          transform: scale(2.4);
+  border-radius: 50%;
+  background-color: #f2f5f8;
+  z-index: -1;
+}
+
+.notfound h1 {
+  font-family: 'Nunito', sans-serif;
+  font-size: 65px;
+  font-weight: 700;
+  margin-top: 0px;
+  margin-bottom: 10px;
+  color: #151723;
+  text-transform: uppercase;
+}
+
+.notfound h2 {
+  font-family: 'Nunito', sans-serif;
+  font-size: 21px;
+  font-weight: 400;
+  margin: 0;
+  text-transform: uppercase;
+  color: #151723;
+}
+
+.notfound p {
+  font-family: 'Nunito', sans-serif;
+  color: #999fa5;
+  font-weight: 400;
+}
+
+.notfound a {
+  font-family: 'Nunito', sans-serif;
+  display: inline-block;
+  font-weight: 700;
+  border-radius: 40px;
+  text-decoration: none;
+  color: #388dbc;
+}
+
+@media only screen and (max-width: 767px) {
+  .notfound .notfound-404 {
+    width: 110px;
+    height: 110px;
+  }
+  .notfound {
+    padding-left: 15px;
+    padding-right: 15px;
+    padding-top: 110px;
+  }
+}
+</style>
+</head>
+
+<body>
+
+<div id="notfound">
+<div class="notfound">
+<div class="notfound-404"></div>
+<h1>404</h1>
+<h2>Oops! Page Not Be Found</h2>
+<p>Sorry but the page you are looking for does not exist, have been removed. name changed or is temporarily unavailable.</p>
+<a href="https://home-assistant.io/">Back to homepage</a>
+</div>
+</div>
+
+</body><!-- This templates was made by Colorlib (https://colorlib.com) -->
```

Comment:
Are we allowed to use this template? Is the license matching for our project?

Reply:
It says

You are allowed to use templates for personal and business usage.
You are allowed to customize templates however you like.
You are allowed to convert/port HTML templates for use for any CMS.
You are allowed to share and distribute them under Colorlib brand name.

This is from there site https://colorlib.com/wp/licence/

Reply:
As far as I can tell yes we can use it.

Reply:
Sure, but the copyright needs to stay intact. As in, this is not a fully open-source template, as it comes with additional licensing (that btw doesn't match the license they pretend to have).

Nevertheless, a simple error in a contribution, later on, can cause problems. Additionally, I don't think this matches our CLA (which you have signed for this PR): <https://www.home-assistant.io/developers/cla/>

Alternatively, we could create a 404.markdown file in the root and render that in our own, existing, layout instead?

Reply:
Sure I will have a playaround with markdown.  I will see what I can come up with.

Reply:
Done.